plugins/scaffolder-backend/api-report.md

```diff
@@ -431,6 +434,10 @@ export interface TaskSpecV1beta2 {
   apiVersion: 'backstage.io/v1beta2';
   // (undocumented)
   baseUrl?: string;
+  // Warning: (ae-forgotten-export) The symbol "TemplateMetadata" needs to be exported by the entry point index.d.ts
```

Comment:
Let's export `TemplateMetadata` to avoid this warning, and also perhaps use it for the `ActionContext` type too?

Reply:
Done.

Reply:
Not quite, needs to be exported all the wait out 😅. Basically need to make sure this isn't adding any new warnings to the API report, while ofc following https://backstage.io/docs/architecture-decisions/adrs-adr004

Reply:
Bonus ping @Joonpark13, since this was marked as resolved and a bit hidden 😅

Reply:
Thanks for surfacing this, I think I've done it correctly now.